bedrock/base/templates/base-resp.html

```diff
@@ -7,6 +7,9 @@
   <head>
     <meta charset="utf-8">{# Note: Must be within first 512 bytes of page #}
 
+    {% javascript 'site' %}
+    {% block newrelic %}{% endblock %}
```

Comment:
It was [recommended in the docs](https://docs.newrelic.com/docs/browser/new-relic-browser/installation-configuration/adding-apps-new-relic-browser#copy-paste-app). Not sure how important it is.